microk8s-resources/wrappers/microk8s-enable.wrapper

```diff
@@ -24,15 +24,7 @@ exit_if_no_permissions
 
 result=1
 for addon in "$@"; do
-  # Making sure the cluster is up and running befor each addon
-  STATUS=$(${SNAP}/microk8s-status.wrapper --wait-ready --timeout 30)
```

Comment:
Could you add the arguments "--wait-ready --timeout 30" to the microk8s-status.wrapper call below so we wait for at most 30 seconds for any service restarts caused by enabling addons?

Reply:
done